spark/dl/src/main/scala/com/intel/analytics/bigdl/example/keras/Utils.scala

```diff
@@ -0,0 +1,156 @@
+/*
```

Comment:
it's too much for a simple example. Maybe what we need to do is simply create a `KerasLenetModel` and then integrate it with the existing `LeNet` instead of duplicating codes. Let's think about this again.

Reply:
Yeah. I just first put it here to make the example runnable for the training = =.

Reply:
But I find `lenet` and `lenetlocal` have almost the same duplicate code for this `Utils`...